examples-server/serve.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+import fastifyStatic from '@fastify/static'
```

Comment:
It'd probably be better to use a static-site framework reflecting on this, would just involve getting Nunjucks into it...

Reply:
I think that a sensible idea

Reply:
Yeah, I agree. We can come back to it after doing a few components. Could be it's easy to do with Vite as well